source/index.md

```diff
@@ -600,9 +602,41 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required.
+Please refer to the [official documentation][GooglePay-PaymentCryptography].
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:]<br />Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
+  <dd>[:base64:]<br />The shared secret derived from the ephemeral public key and
+    your private key.</dd>
+  <dt>googlepay[recipient_id]</dt>
+  <dd>merchant:[:merchant_id:]<br />The ID of the merchant preprended with "merchant:".</dd>
+</dl>
+
+<p class="alert alert-info">
+  <b>Notice:</b> Signing is required to use the <code>googlepay</code> payment
+  method.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> may be fully
+  3-D Secured, 3-D Secure attempted, or with no 3-D Secure; this is indicated by
+  the <code>eciIndicator</code> of the <code>paymentMethodDetails</code>.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> cannot be a
+  subsequent recurring authorization.
```

Comment:
Perhaps add:

> If marked as `recurring`, the authorization will be an initial recurring authorization.

It would be appropriate to notice that it'd be a "CIT" (Cardholder Initiated Transaction), but I do not thing it's appropriate to add in this PR. It should be done throughout the documentation (thus, in a separate PR) 🙂